doc/source/workflows/basics.rst

```diff
@@ -362,6 +370,8 @@ The following example shows how to implement the recursive ``factorial`` program
         if n == 1:
             return 1
         else:
+            # Here a DAG is passed to the continuation.
+            # The DAG will continue to be executed after this task.
             return workflow.continuation(multiply.bind(n, factorial.bind(n - 1)))
```

Comment:
In line 408 below:
return workflow.continuation(finalize_or_cancel.bind([f1, f2], [hotel]))

How are the semantics different if you just had:
return finalize_or_cancel.bind([f1, f2], [hotel])

Reply:
We don't allow return a DAG directly. Although right now there is no difference in workflow engine, in ray core engine (`g.bind().execute()`) it's not allowed.
The direction should be that DAG is just a declarative DSL and it should be able to be executed in both workflow and ray core engine.